sdk/core/azure-core-metrics-opentelemetry/src/main/java/com/azure/core/metrics/opentelemetry/OpenTelemetryAttributes.java

```diff
@@ -23,7 +41,13 @@ class OpenTelemetryAttributes implements TelemetryAttributes {
             Objects.requireNonNull(kvp.getKey(), "'key' cannot be null.");
             Objects.requireNonNull(kvp.getValue(), "'value' cannot be null.");
 
-            OpenTelemetryUtils.addAttribute(builder, kvp.getKey(), kvp.getValue());
+            // TODO: by GA we need to figure out default naming (when no mapping is defined)
+            // and follow otel attributes conventions if we can or make sure mapping is defined
+            // for all attributes
```

Comment:
We should probably tackle this ~now?

Reply:
curious what's the urgency?

this mapping exists in beta metrics package with no requirement to GA in the nearest future. 
However, we have an ask to GA tracing plugin in Zn, where we'll do exactly the same mapping and we'll figure it out there first. Assuming we'll resource the ask, it should happen in the next few months.

And I'd really love to take it now, but we're working with OTel community on changing these exact attributes, so anything I could write might change very soon.